### DISCUSSION TITLE
Incompatibility between spacy versions for English and Spanish models

### DISCUSSION THREAD

**Discussion by Catalina1100** [explosion/spaCy#8925] [Help: Installation] [answered]
Hey, guys!

I was trying to work with the English and Spanish models in Google Collab, but ran into errors.

It seems that the English model requires spacy versions 2.3.1 or 2.3.5. And the Spanish model supports only version 3.1. So any time I change the language I need to reinstall a different version of spacy.  Is there any common version for both languages? Did I miss anything?

Cheers,
Kateryna

**Comment by svlandeg** [comment, marked as answer]
Hi Kateryna,

spaCy versions v2 and v3 are definitely quite different, so you'll want to pick one and stick to that. For each of our major version releases, we've trained and released specific models for each of the languages, so you'll have to make sure that you've downloaded the correct models for the spaCy version you want to use.

You can double check by running `python -m spacy validate`, which will also provide you with the right command to update those models that are not compatible.

Let us know if you run into any issues with that!

**Reply by Catalina1100** [reply]
Dear Sofie,

Got it! Thanks for the quick reply:)
